src/sentry/workflow_engine/tasks/workflows.py

```diff
@@ -93,14 +93,14 @@ def process_workflow_activity(activity_id: int, group_id: int, detector_id: int)
     on_silent=DataConditionGroup.DoesNotExist,
 )
 def process_workflows_event(
-    project_id: int,
     event_id: str,
     group_id: int,
     occurrence_id: str | None,
     group_state: GroupState,
     has_reappeared: bool,
     has_escalated: bool,
     start_timestamp_seconds: float | None = None,
+    project_id: int | None = None,
```

Comment:
looks like we could omit this here now too 🎉  `--dependencies;`

Reply:
in the next PR!